### ISSUE TITLE
RDKit fails to import Chem module on Mac OSX - Version 

### ISSUE THREAD

**Issue by leelasd** [rdkit/rdkit#1175]
In RDKit version:2016.03.4  and np111py27_1   

I tried to import Chem from rdkit. It crashed and gave me the error below. Older version `2016.03.3` was working fine. 

Any ideas why this is happening ? It works fine on Linux machine. 

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-4-17d7acb427ab> in <module>()
----> 1 from rdkit import Chem

/Users/Sriram/anaconda/lib/python2.7/site-packages/rdkit/Chem/__init__.py in <module>()
     21 from rdkit import DataStructs
     22 from rdkit.Geometry import rdGeometry
---> 23 from rdkit.Chem import PeriodicTable as pyPeriodicTable
     24 from rdkit.Chem import rdchem
     25 _HasSubstructMatchStr=rdchem._HasSubstructMatchStr

ImportError: cannot import name PeriodicTable
```

**Comment by greglandrum** [comment]
That is an unusual one.
Can you please look in '/Users/Sriram/anaconda/lib/python2.7/site-packages/rdkit/Chem' to see if there is a PeriodicTable.py in it?

**Comment by leelasd** [comment]
Yes it is there !! Dont know why this happened. Now I get a different error. It looks like this

```
In [1]: import rdkit

In [2]: from rdkit import Chem
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-17d7acb427ab> in <module>()
----> 1 from rdkit import Chem

/Users/Sriram/anaconda/lib/python2.7/site-packages/rdkit/Chem/__init__.py in <module>()
     27 from rdkit.Chem.rdmolfiles import *
     28 from rdkit.Chem.rdmolops import *
---> 29 from rdkit.Chem.inchi import *
     30 
     31 def QuickSmartsMatch(smi,sma,unique=True,display=False):

/Users/Sriram/anaconda/lib/python2.7/site-packages/rdkit/Chem/inchi.py in <module>()
     35 import logging
     36 
---> 37 from rdkit.Chem import rdinchi
     38 from rdkit import RDLogger
     39 logger = RDLogger.logger()

ImportError: dlopen(/Users/Sriram/anaconda/lib/python2.7/site-packages/rdkit/Chem/rdinchi.so, 2): Library not loaded: @rpath/libInchi.1.dylib
  Referenced from: /Users/Sriram/anaconda/lib/python2.7/site-packages/rdkit/Chem/rdinchi.so
  Reason: Incompatible library version: rdinchi.so requires version 1.0.0 or later, but libInchi.1.dylib provides version 0.0.0
```

**Comment by greglandrum** [comment]
Do you have either DYLD_LIBRARY_PATH or DYLD_FALLBACK_LIBRARY_PATH set?

It's probably easiest if you just post the output you get when you type "export" in the terminal from which you launch Python

**Comment by annademidova** [comment]
Hey I'm struggling with the same issue. Where can I edit ?

Here's the output of "export":

declare -x Apple_PubSub_Socket_Render="/private/tmp/com.apple.launchd.zgw677CmQU/Render"
declare -x CONDA_DEFAULT_ENV="my-rdkit-env"
declare -x CONDA_PATH_BACKUP="/Users/annademidova/miniconda2/bin:/Users/annademidova/anaconda2/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
declare -x CONDA_PREFIX="/Users/annademidova/miniconda2/envs/my-rdkit-env"
declare -x CONDA_PS1_BACKUP="\\h:\\W \\u\\\$ "
declare -x HOME="/Users/annademidova"
declare -x LANG="en_US.UTF-8"
declare -x LOGNAME="annademidova"
declare -x OLDPWD
declare -x OMF_CONFIG="/Users/annademidova/.config/omf"
declare -x OMF_PATH="/Users/annademidova/.local/share/omf"
declare -x PATH="/Users/annademidova/miniconda2/envs/my-rdkit-env/bin:/Users/annademidova/miniconda2/bin:/Users/annademidova/anaconda2/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
declare -x PS1="(my-rdkit-env) \\h:\\W \\u\\\$ "
declare -x PWD="/Users/annademidova"
declare -x SHELL="/usr/local/bin/fish"
declare -x SHLVL="2"
declare -x SSH_AUTH_SOCK="/private/tmp/com.apple.launchd.vKkIwogKxU/Listeners"
declare -x TERM="xterm-256color"
declare -x TERM_PROGRAM="Apple_Terminal"
declare -x TERM_PROGRAM_VERSION="400"
declare -x TERM_SESSION_ID="6CEEECA1-CB93-47A8-BA74-2F5D8A43EA83"
declare -x TMPDIR="/var/folders/8t/lnpx5t_s4dlcqkk844ywvbs40000gn/T/"
declare -x USER="annademidova"
declare -x XPC_FLAGS="0x0"
declare -x XPC_SERVICE_NAME="0"
